Classes/Domain/Repository/PeriodRepository.php

```diff
@@ -25,6 +24,13 @@ class PeriodRepository extends Repository
         'end' => QueryInterface::ORDER_ASCENDING
     ];
 
+    public function initializeObject()
+    {
+        $querySettings = $this->objectManager->get(Typo3QuerySettings::class);
+        $querySettings->setRespectStoragePage(false);
```

Comment:
Für das komplette Repo?!?! Das gilt direkt für alle zukünftigen Methoden, die hier noch reinkommen. Bist Du Dir hier wirklich sicher? Ansonsten bitte individuell in die Methode einbauen

Reply:
Es werden sehr spezifische Fälle kommen müssen in denen wir die Storage Page wieder beachten möchten. Deshalb habe ich das ganze jetzt mal für das gesamte Repo gesetzt, damit ich es nicht in jeder Methode einzeln deaktiviere.